packages/fints/package.json

```diff
@@ -75,6 +75,7 @@
     "dependencies": {
         "bind-decorator": "^1.0.11",
         "date-fns": "^1.29.0",
+        "fast-xml-parser": "^3.12.12",
```

Comment:
You added this library but I can't see the yarn.lock file to have changed. Can you please commit it as well?